src/sagemaker/tensorflow/README.rst

```diff
@@ -614,188 +612,25 @@ Note that TensorBoard is not supported when passing wait=False to ``fit``.
 Deploying TensorFlow Serving models
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After a ``TensorFlow`` Estimator has been fit, it saves a ``TensorFlow Serving`` model in
-the S3 location defined by ``output_path``. You can call ``deploy`` on a ``TensorFlow``
+After a TensorFlow Estimator has been fit, it saves a TensorFlow ``SavedModel`` in
```

Comment:
I don't think "Estimator" needs to be capitalized here

Reply:
ok